README.md

```diff
@@ -105,77 +37,54 @@ To build all packages run:
 $ npm run build
 ```
 
-### cmcd-validator-library
+### [cmcd-validator-library](https://github.com/montevideo-tech/cmcd-validator/tree/develop/packages/cmcd-validator-library)
 
 To build only the library run:
 ```bash
 $ npm run build-library
 ```
+You can find the npm 
 
-### cmcd-validator-demo-app
+### [cmcd-validator-demo-app](https://github.com/montevideo-tech/cmcd-validator/tree/develop/packages/cmcd-validator-demo-app)
 
-To lunch the demo app first you need to build the **cmcd-validator-library** and then  lunch the following command:
+
+To launch the demo app first you need to build the **cmcd-validator-library** and then  launch the following command:
```

Comment:
I think it should be run the following command instead of launch the following..